src/style.css

```diff
@@ -1 +1,78 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body,
+html {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  /* padding: 10px 50px; */
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+
+  color: #000;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo a {
```

Comment:
Don't use several selectors where you can use one class

Reply:
fix it everywhere

Reply:
fixed

Reply:
except one where I needed to select a specific element without adding a class to it.